<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2560</width>
    <height>1440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QOpenGLWidget" name="openGLWidget_main">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>691</width>
      <height>561</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QTreeView" name="treeView_instance">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>60</y>
      <width>211</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>609</y>
      <width>531</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QPushButton" name="btn_createObject">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>创建物体</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="btn_createLight">
       <property name="text">
        <string>创建灯光</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QPushButton" name="btn_switchCamera">
       <property name="text">
        <string>切换摄像机</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QPushButton" name="btn_createCamera">
       <property name="text">
        <string>创建摄像机</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QComboBox" name="comboBox_light">
       <item>
        <property name="text">
         <string>点光源</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>方向光源</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>面光源/聚光灯</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QComboBox" name="comboBox_object">
       <item>
        <property name="text">
         <string>立方体</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>经纬球</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>圆环</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget_attribute">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>410</y>
      <width>411</width>
      <height>471</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::TabPosition::West</enum>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab_transform">
     <attribute name="title">
      <string>变换</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea_transform">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>381</width>
        <height>450</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="scrollAreaTransWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>350</width>
         <height>450</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_material">
     <attribute name="title">
      <string>材质</string>
     </attribute>
     <attribute name="toolTip">
      <string>物体材质属性</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea_material">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>351</width>
        <height>441</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="scrollAreaMaterialWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>345</width>
         <height>439</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_light">
     <attribute name="title">
      <string>光源</string>
     </attribute>
     <attribute name="toolTip">
      <string>光源属性</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea_light">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>311</width>
        <height>650</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="scrollAreaLightWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>339</width>
         <height>900</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_camera">
     <attribute name="title">
      <string>摄像机</string>
     </attribute>
     <attribute name="toolTip">
      <string>摄像机属性</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea_camera">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>341</width>
        <height>430</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="scrollAreaCameraWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>340</width>
         <height>410</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_texture">
     <property name="toolTip">
      <string>物体数据属性</string>
     </property>
     <property name="whatsThis">
      <string>物体数据属性</string>
     </property>
     <attribute name="title">
      <string>物体贴图</string>
     </attribute>
     <attribute name="toolTip">
      <string>物体数据属性</string>
     </attribute>
     <widget class="QPushButton" name="btn_addUVTexture">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>添加贴图</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_RemoveUVTexture">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>移除贴图</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>切换顶点匹配</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_textureType">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>漫反射贴图</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>法线贴图</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>立方体贴图</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>HDR环境贴图</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2560</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionBias"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>渲染</string>
    </property>
    <addaction name="actionRenderImage"/>
    <addaction name="actionRenderAnimation"/>
    <addaction name="actionRenderSound"/>
    <addaction name="actionLockUi"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>窗口</string>
    </property>
    <addaction name="actionNewWindow"/>
    <addaction name="actionNewMainWindow"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionTutorial"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>导入</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>导出</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>重做</string>
   </property>
  </action>
  <action name="actionBias">
   <property name="text">
    <string>偏好设置</string>
   </property>
  </action>
  <action name="actionRenderImage">
   <property name="text">
    <string>渲染图像</string>
   </property>
  </action>
  <action name="actionRenderAnimation">
   <property name="text">
    <string>渲染动画</string>
   </property>
  </action>
  <action name="actionRenderSound">
   <property name="text">
    <string>渲染音频</string>
   </property>
  </action>
  <action name="actionLockUi">
   <property name="text">
    <string>锁定界面</string>
   </property>
  </action>
  <action name="actionNewWindow">
   <property name="text">
    <string>新建窗口</string>
   </property>
  </action>
  <action name="actionNewMainWindow">
   <property name="text">
    <string>新建主窗口</string>
   </property>
  </action>
  <action name="actionTutorial">
   <property name="text">
    <string>教程</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>保存为...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
